llpc/test/shaderdb/core/OpTypeSampledImage_TestWaterfallScalarize.frag

```diff
@@ -0,0 +1,29 @@
+#version 450
+#extension GL_EXT_nonuniform_qualifier : require
+
+layout(set = 0, binding = 7) uniform sampler2D _11[];
+
+layout(location = 0) out vec4 _3;
+layout(location = 3) flat in int _4;
+layout(location = 1) flat in vec2 _6;
+
+void main()
+{
+    int _12 = _4;
+    _3 = texture(_11[nonuniformEXT(_12)], _6);
+}
+
+// BEGIN_SHADERTEST
+/*
+; RUN: amdllpc -scalarize-waterfall-descriptor-loads -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; Make sure that there is a single begin index
+; Make sure that there is a single waterfall.readfirstlane for the offset
```

Comment:
Can you move the test description to the top of the file? This way it's easier to tell what tests do.